src/Fluid/Flu_FixUp_Restrict.cpp

```diff
@@ -91,7 +91,11 @@ void Flu_FixUp_Restrict( const int FaLv, const int SonFluSg, const int FaFluSg,
 #  else
    const bool ResMag  = false;
 #  endif
-   const int PS1_half = PS1 / 2;
+#  ifndef _DEDT_NU
+   const long _DEDT_NU = 0;
+#  endif
+   const long ResVar   = _TOTAL - _DEDT_NU;
```

Comment:
This variable needs a better name to match its usage. How about `EoSVar` since it is mainly used for ensuring the EoS consistency?

Reply:
True. Updated.